src/box/error.h

```diff
@@ -122,6 +122,8 @@ box_error_clear(void);
 /**
  * Set the last error.
  *
+ * \param file
+ * \param line
```

Comment:
It suppresses the following Doxygen warning:

> <...>/src/module.h:2670: warning: The following parameters of box_error_set(const char *file, unsigned line, uint32_t code, const char *format,...) are not documented:
>   parameter 'file'
>   parameter 'line'

I wouldn't say that the parameters are documented now. My proposal:

```c
 * \param file file name, usually __FILE__ macro
 * \param line line number in the file, usually __LINE__ macro
```

Reply:
Updated, thanks